lib/matplotlib/widgets.py

```diff
@@ -1930,61 +1930,59 @@ def toggle_selector(event):
     _shape_klass = Rectangle
 
     def __init__(self, ax, onselect, drawtype='box',
-                 minspanx=None, minspany=None, useblit=False,
```

Comment:
There's a check for `self.minspanx is not None`, so I'm not sure if this should change.

Reply:
that's fine, if it's None then the test `spanx < minspanx` is skipped, but if it's zero then `spanx < minspanx` is never true, so the end result is the same.